util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialConfiguration.java

```diff
@@ -187,12 +182,8 @@ public static CredentialFactory getCredentialFactory(
                 TOKEN_SERVER_URL_SUFFIX.withPrefixes(keyPrefixes).get(config, config::get))
             .setProxyAddress(
                 PROXY_ADDRESS_SUFFIX.withPrefixes(keyPrefixes).get(config, config::get))
-            .setProxyUsername(
-                RedactedString.create(
-                    PROXY_USERNAME_SUFFIX.withPrefixes(keyPrefixes).getPassword(config)))
-            .setProxyPassword(
-                RedactedString.create(
-                    PROXY_PASSWORD_SUFFIX.withPrefixes(keyPrefixes).getPassword(config)))
+            .setProxyUsername(PROXY_USERNAME_SUFFIX.withPrefixes(keyPrefixes).getPassword(config))
```

Comment:
overall the implementation looks good. I am just wondering if we need to redact username? printing out username sometimes is helpful. Looking at the code, I do see that it was redacted earlier. Just wondering if if is really needed.

Reply:
This is already redacted `getPassword` returns redacted string